docs/projects/ad7124_asdz/index.rst

```diff
@@ -0,0 +1,148 @@
+.. _ad7124:
```

Comment:
The name doesn't match the pattern
```suggestion
.. _ad7124_asdz:
```

Reply:
Fixed!

Reply:
This is a project having only a SPI interface